Dockerfile

```diff
@@ -1,17 +1,11 @@
 FROM dmscid/lewis-depends:latest
 
-# Copying this separately from the rest of lewis allows the
-# pip install step to be cached until the requirements change.
-# Rebuilding is faster and doesn't require an internet connection.
-COPY requirements.txt /lewis/
+# Lewis source is pulled in from current directory (note .dockerignore filters out some files)
+COPY . /lewis
 
 # Install any missing PIP requirements, clear PIP cache, and delete all compiled Python objects
-RUN pip install -r lewis/requirements.txt && \
+RUN pip install -e /lewis && \
```

Comment:
Thanks for clarifying this, I wasn't entirely sure about the implications about this. The faster build time doesn't seem like an enormous price to pay, but the "doesn't require an internet connection" sounds like it's introducing a slight inconvenience.

What about moving those into `lewis-depends`?